lang/main.json

```diff
@@ -75,6 +75,10 @@
             "content": "Learn about your users through easy integration with Piwik, Google Analytics, and other usage monitoring and statistics systems."
         }
     },
+    "startupoverlay": {
+        "policyText": "&nbsp;",
+        "title": "Jitsi Meet needs to use your microphone and camera."
+    },
```

Comment:
Could you please, replace Jitsi Meet with __app__, and use the post-process, or pass the variable to the translateElement options getting it from interfaceConfig.APP_NAME (like dialog.thankYou).